docs/how-tos/one-to-many.md

```diff
@@ -125,10 +125,17 @@ You can click the **{}** button to switch to advanced view and see the changes.
 
 Don't forget to **Save**.
 
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Check your machine's logs**
+
+After configuring fragment modifications, check your machine's [**LOGS** tab](/cloud/machines/#logs).
```

Comment:
[opt] This is a place where I think either makes sense. What I meant with my other comment is that I think it makes sense to either day "modify a fragment" or "overwrite a portion of a fragment." I just think the confusing thing is if we say "overwrite a fragment."
```suggestion
After configuring fragment overwrites, check your machine's [**LOGS** tab](/cloud/machines/#logs).
```